api/src/main/java/org/openmrs/module/kenyaemr/reporting/library/MOH706/Moh706IndicatorLibrary.java

```diff
@@ -27,13 +27,13 @@ public Moh706IndicatorLibrary(Moh706LabCohortLibrary moh706LabCohortLibrary) {
 
 	public CohortIndicator getTotalTestsByConcept(Integer labSetConceptId) {
 		return cohortIndicator(
-			"All patients who have urinalysis glucose",
+			"All patients who have tests",
```

Comment:
You can consider qualifying which tests

Reply:
Am passing tests as parameters  - what did you mean by qualify